spec/lib/api/v3/support/link_examples.rb

```diff
@@ -49,6 +52,20 @@
   end
 end
 
+shared_examples_for 'has an untitled action link' do
+  it_behaves_like 'has an untitled link'
+
+  it 'indicates the desired method' do
+    is_expected.to be_json_eql(method.to_json).at_path("_links/#{link}/method")
+  end
+
+  describe 'without permission' do
+    let(:permissions) { all_permissions - [permission] }
```

Comment:
Yes.
I'd argue that this is the point of using this example instead of `has an untitled link`, which is also a valid choice...